src/Libplanet.Action/ActionContext.cs

```diff
@@ -69,6 +72,9 @@ internal class ActionContext : IActionContext
             ? _txs
             : ImmutableList<ITransaction>.Empty;
 
+        /// <inheritdoc cref="IActionContext.Evidences"/>
+        public IReadOnlyList<Evidence> Evidences { get; }
```

Comment:
Just `Evidence` stands for both plural and singular.

Reply:
There are a lot of `evidences` that are used as identifiers (field, property). Should I rename them all to evidence?

Reply:
IMO It is not a big deal when using `Find and replace`...

Reply:
By the way, the namespace is `Evidences`. How should this be handled?

Reply:
Change needed I guess..?

Reply:
Then it will overlap with the `Evidence` class.

Reply:
- Changed namespace from `Evidences` to `Evidence`.
- Changed class name from `Evidence` to `EvidenceBase`.
- Changed key (used for serialisation, like bencode) from `evidences..` to `evidence..`.